release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -2902,6 +2902,8 @@ To avoid this issue, you can enable confidential VMs on an existing cluster by u
 
 * Installation fails when installing {product-title} with the `bootMode` set to `UEFISecureBoot` on a node where Secure Boot is disabled. Subsequent attempts to install {product-title} with Secure Boot enabled will proceed normally. (link:https://issues.redhat.com/browse/OCPBUGS-19884)[*OCPBUGS-19884*])
 
+* In {product-title} {product-version}, a `MachineConfig` with Ignition version 3.4 might fail scans of the `api-collector` pods with `CrashLoopBackOff` errors, causing the Compliance Operator to not work as expected. (link:https://issues.redhat.com/browse/OCPBUGS-18025[*OCPBUGS-18025*])
```

Comment:
Consider using either a [general reference](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#general-references) or use an [object reference](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#object-references) for object names.
```suggestion
* In {product-title} {product-version}, a `MachineConfig` object with Ignition version 3.4 might fail scans of the `api-collector` pods with `CrashLoopBackOff` errors, causing the Compliance Operator to not work as expected. (link:https://issues.redhat.com/browse/OCPBUGS-18025[*OCPBUGS-18025*])
```